packages/hebao_v2/contracts/lib/EIP712.sol

```diff
@@ -37,15 +38,19 @@ library EIP712
     }
 
     function hashPacked(
-        bytes32 domainSeperator,
-        bytes   memory encodedData
+        bytes32 domainHash,
```

Comment:
maybe stick to `domainSeperator` because when you call this method, you passed in `DOMAIN_SEPARATOR`